### PR TITLE
Fixes typo on Getting started page

### PR DESCRIPTION
**Pull request by Pr0x1m4**
There's a typo on Getting Started Page as well as the Module Initialization scrollbar didnt align (drove me nuts), this should fix it.

### CODE REVIEW

**Comment by Pr0x1m4** [comment]
Any ideas as to why CI is building this PR?

File: index.html [hunk 1]
```diff
@@ -137,7 +137,15 @@ <h2 class="minitext">
   define("three", function() { return require("two") + require("one"); });
   define("four", function() { return require("three") + require("one"); });
 })();
-three = require("three");</code></pre>
+three = require("three");
```
Comment (TheSavior) [inline index.html]: Is this change necessary?

Comment (Pr0x1m4) [inline index.html]: It aligns the scrollbar to the end of the code block, see before and after.
2017-05-03-205103_3840x2160_scrot
2017-05-03-205206_3840x2160_scrot

Comment (Pr0x1m4) [inline index.html]: Not sure how else to accomplish this

Comment (TheSavior) [inline index.html]: Got it. This should be done with css to make all examples on the site the same. We should handle that separately from this PR. Can you undo these blank lines?

Comment (Adriana1999) [inline index.html]: In German
